### PR TITLE
Add Quectel EC20 4G LTE Module USB CDC/ACM support

### PR DESCRIPTION
**Pull request by aroma890225**
Add EC20 VID&PID to USB HOST CDC ACM Driver.

1. /dev/ttyACM0 is the NMEA Port,

2. /dev/ttyACM1 is the AT Port,

3. /dev/ttyACM2 is the Modem Port.

   Completed PPP dial-up test on /dev/ttyACM1.
   
    avatar

### CODE REVIEW

File: boards/arm/imxrt/imxrt1060-evk/src/imxrt_usbhost.c [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+  /* Make sure we don't accidentially switch on USB bus power */
+
+  *((uint32_t *)IMXRT_USBNC_USB_OTG1_CTRL) = USBNC_PWR_POL;
+  *((uint32_t *)0x400d9030)                = (1 << 21);
+  *((uint32_t *)0x400d9000)                = 0;
+
+  /* Setup pins, with power initially off */
+
+//  imxrt_config_gpio(GPIO_USBOTG_PWR);
```
Comment (acassis) [inline boards/arm/imxrt/imxrt1060-evk/src/imxrt_usbhost.c]: Please remove C++ comments from the source code. Did you check the modified files against nxstyle? nxstyle -m 86 filename.c/h

**Comment by acassis** [comment]
Ok, I changed my mind. I'm fix these issues and will merge it.

File: boards/arm/imxrt/imxrt1060-evk/src/imxrt_usbhost.c (hunk 1 above)
Comment (aroma890225) [reply boards/arm/imxrt/imxrt1060-evk/src/imxrt_usbhost.c]: Sorry about that. I will check the code style carefully next time.

Comment (acassis) [reply boards/arm/imxrt/imxrt1060-evk/src/imxrt_usbhost.c]: No problem! Please let me know if it is working with the code integrated on mainline!

**Pull request merged**